src/routes/libraries.spec.js

```diff
@@ -523,49 +583,45 @@ describe('/libraries', () => {
         describe('Providing search fields that are invalid (?search=backbone.js&search_fields=this-field-doesnt-exist)', () => {
             // If invalid fields make it to Aloglia, it will error, so this tests that we're filtering them first
             const path = '/libraries?search=backbone.js&search_fields=this-field-doesnt-exist';
-            const test = () => request().get(path);
+            const fetch = () => request(path);
+
+            // Fetch the endpoint
             let response;
-            before('fetch endpoint', done => {
-                fetch(test, 5000).then(res => {
-                    response = res;
-                    done();
-                });
-            });
+            before('fetch endpoint', () => fetch().then(res => { response = res; }));
+
+            // Test the endpoint
             testCors(path, () => response);
-            it('returns the correct Cache headers', done => {
+            it('returns the correct Cache headers', () => {
                 expect(response).to.have.header('Cache-Control', 'public, max-age=21600'); // Six hours
-                done();
             });
-            it('returns a JSON body with \'results\', \'total\' and \'available\' properties', done => {
+            it('returns the correct status code', () => {
+                expect(response).to.have.status(200);
+            });
+            it('returns a JSON body with \'results\', \'total\' and \'available\' properties', () => {
                 expect(response).to.be.json;
                 expect(response.body).to.have.property('results').that.is.an('array');
                 expect(response.body).to.have.property('total').that.is.a('number');
                 expect(response.body).to.have.property('available').that.is.a('number');
-                done();
             });
-            it('returns all available hits', done => {
+            it('returns all available hits', () => {
                 expect(response.body.results).to.have.lengthOf(response.body.total);
                 expect(response.body.results).to.have.lengthOf(response.body.available);
-                done();
             });
             describe('Library object', () => {
-                it('is an object with \'name\' and \'latest\' properties', done => {
+                it('is an object with \'name\' and \'latest\' properties', () => {
                     for (const result of response.body.results) {
                         expect(result).to.have.property('name').that.is.a('string');
                         expect(result).to.have.property('latest').that.is.a('string');
                     }
-                    done();
                 });
                 // This is fragile! backbone.js is the most popular package in this query so we should see it first
```

Comment:
Looking at this test, we don't need to actually search for a specific library at all -- if I remove the protection that this tests, it will error with just search_fields passed to Algolia, no search required.